packages/components/src/SVGIcon/SVGIcon.stories.tsx

```diff
@@ -49,34 +49,42 @@ type Args = React.ComponentProps<typeof SVGIcon> & {
 
 const Template: Story<Args> = ({ icon, color, ...rest }) => {
   const Icon = allIcons[icon];
-  const computedColor = Tokens[color];
+  const computedColor = color && Tokens[color];
   return <Icon {...rest} color={computedColor} />;
 };
 
 const defaultArgs = {
   icon: Object.keys(allIcons)[0],
-  role: "presentation",
+  role: "presentation" as const,
 };
 
 export const Small = Template.bind(null);
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore todo: figure out SVGIcon types
```

Comment:
I think the SvgIcon types will take some investigation to figure out what's up and how we should write them. Opted to punt on that for from this PR.